go.mod

```diff
@@ -90,3 +91,10 @@ replace (
 	k8s.io/metrics => k8s.io/metrics v0.0.0-20190918202012-3c1ca76f5bda
 	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.0.0-20190918201353-5cc279503896
 )
+
+replace (
+	github.com/openshift/api => github.com/openshift/api v3.9.1-0.20190717200738-0390d1e77d64+incompatible
```

Comment:
Bumping OLM to 0.12.0 resulted in go not being able to resolve these dependencies. Pinning them with replace fixed that issue. I'm going to test this PR with the previous version of OLM and operator-registry to see if I can remove this `replace` block. If it does not work with those older versions I will add comments to this block explaining why it is here.